README.md

```diff
@@ -32,8 +32,22 @@ Bumping patch for version 1.2.4
 v1.2.3
 v1.2.4
 v1.2.5
+
+> gitversion --prefix v show
+v1.2.5
 ```
 
+### Auto
+
```

Comment:
I think an example of its application would be helpful.
e.g.

Example commit message:
> "Major change, crazy things happened"

```
gitversion -v "> gitversion --prefix v bump auto
Bumping major for version 1.2.4
2.0.0
```